### DISCUSSION TITLE
Create UI

### DISCUSSION THREAD

**Discussion by SuMyaing2** [keploy/keploy#226] [Q&A]
create UI automation test following this step - open https://shopee.co.th/ 
- search for iPhone - select sort price from desc - validate TOP 5 items sort by price correctly

**Comment by slayerjain** [comment]
@SuMyaing2 I'm not sure if I understood the question.  Keploy doesn't do UI testing. I think selenium IDE might be something which can help record UI test cases.

**Reply by SuMyaing2** [reply]
Thank you for your answer 🙏

**Reply by slayerjain** [reply]
> Thank you for your answer 🙏

I’m sorry I meant that we do API tests and not UI tests. The original comment is also updated to avoid confusion.
